src/Feature/Buffers/Feature_Buffers.re

```diff
@@ -441,6 +512,47 @@ module Effects = {
       }
     });
   };
+
+  let previewFileInEditor =
+      (
+        ~font: Service_Font.font,
+        ~languageInfo: Exthost.LanguageInfo.t,
+        ~split=`Current,
+        ~position=None,
+        ~grabFocus=true,
+        ~filePath,
+        model,
+      ) => {
+    Isolinear.Effect.createWithDispatch(
+      ~name="Feature_Buffers.openFileInEditor", dispatch => {
```

Comment:
"openFileInEditor" -> "previewFileInEditor"

Also if its just the name and bool that are different here, maybe remove the duplicated logic?